gui/static/js/components/blockly.js

```diff
@@ -0,0 +1,45 @@
+var React = require('react');
+var ReactDOM = require('react-dom');
+
+export default class Blockly extends React.Component {
+    //TODO getBlocklyScript does not work
```

Comment:
oohh even if it's temporary i'd appreciate if it were made into a github issue so that other people can claim it / know what's going on?